src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -694,13 +697,28 @@ export class PageControllerBase {
     if (feedbackContextInfo) {
       viewModel.name = feedbackContextInfo.formTitle
     }
+
+    let feedbackLink: string | undefined = ''
+
     // setting the feedbackLink to undefined here for feedback forms prevents the feedback link from being shown
     if (this.def.feedback?.url) {
-      viewModel.feedbackLink = this.feedbackUrlFromRequest(request)
+      feedbackLink = this.feedbackUrlFromRequest(request)
```

Comment:
We've got the check for `this.def.feedback?.url` in **this.feedbackUrlFromRequest()** too

Shall we simplify all this?

```js
const { feedback } = this.def

let feedbackLink = this.feedbackUrlFromRequest(request)

if (!feedbackLink) {
  feedbackLink = feedback?.emailAddress
    ? `mailto:${feedback.emailAddress}`
    : config.feedbackLink
}
```

Reply:
Happy to. I'm ditching the `feedbackUrlFromRequest` method entirely as it was doing some stuff we don't need. In my clean up PR to follow https://github.com/DEFRA/forms-runner/pull/228 I'm just using `feedback.url` instead of the needlessly complex `this.feedbackUrlFromRequest(request)`.